.binder/environment.yml

```
@@ -14,12 +15,13 @@ dependencies:
   - twine
   # demo stuff
   - jupyter-server-proxy
-  - jupyterlab >=3.1.0a11,<4
   - retrolab
   # extra docs tools
   - sphinx-autobuild
 
   ### DOCS ENV ###
+  # primary upstream
+  - jupyterlab >=3.1.0a11,<4
```

Looks like this could also be bumped?

```suggestion
  - jupyterlab >=3.1.0b0,<4
```

---

noice